components/backtop/index.js

```diff
@@ -0,0 +1,3 @@
+import BackTop from './back-top.vue'
+
```

Comment:
where is your style file?

Reply:
style scoped

Reply:
so what if antd upgrade style? modify .vue file is not a good idea when just upgrade style.